src/public/modules/forms/admin/directives/validate-form-emails-input.directive.js

```diff
@@ -0,0 +1,39 @@
+'use strict'
+
+const validator = require('validator')
+const get = require('lodash/get')
+
+angular
+  .module('forms')
+  .directive('validateFormEmailsInput', validateFormEmailsInput)
+
+function validateFormEmailsInput() {
+  return {
+    restrict: 'A',
+    require: 'ngModel',
+    link: (scope, _elem, _attr, ngModel) => {
+      ngModel.$validators.validEmails = (rawInput) => {
+        const emails = String(rawInput).split(',')
+        return (
+          get(scope, 'vm.formData.responseMode') !== 'email' ||
```

Comment:
not sure about this, but since `get(scope, 'vm.formData.responseMode') !== 'email'` is repeated, can we initiate it once outside of the validators, or do we need them at the point of evaluation?

Reply:
haha I didn't initiate outside because I wasn't sure, but let me try this now

Reply:
edit: we can fix this in React migration